indico/core/signals/event/registration.py

```diff
@@ -110,3 +110,10 @@
 object being cloned. The new ``RegistrationForm`` object is passed in the ``new_form``
 kwarg.
 ''')
+
+before_registration_email_send = _signals.signal('before_registration_email_send', '''
```

Comment:
Since this signal is registration-specific, the context string isn't really useful and the signal definition should be in `indico/core/signals/event/registration.py`.

How about turning this into a `before_custom_registration_email` signal in there which uses the `registration` as the sender and the `email` kwarg as-is?

Reply:
Sure, if we go with suggestion below, no need this anymore.